http/http-advanced/src/test/java/io/quarkus/ts/openshift/http/AbstractHttpTest.java

```diff
@@ -97,43 +74,49 @@ public void testGrpc() {
     @Test
     @DisplayName("Http/2 Server test")
     public void http2Server() throws InterruptedException, URISyntaxException {
-
-        URL res = getClass().getClassLoader().getResource("META-INF/resources/server.truststore");
-        File file = Paths.get(res.toURI()).toFile();
-        String truststore = file.getAbsolutePath();
-
         CountDownLatch done = new CountDownLatch(1);
-        WebClientOptions options = new WebClientOptions().setProtocolVersion(HttpVersion.HTTP_2).setSsl(true).setVerifyHost(false).setUseAlpn(true).setTrustStoreOptions(new JksOptions().setPassword("password").setPath(truststore));
-        Uni<JsonObject> content = WebClient.create(Vertx.vertx(), options).getAbs(getAppEndpoint() + "/hello")
+        Uni<JsonObject> content = WebClient.create(Vertx.vertx(), defaultVertxHttpClientOptions()).getAbs(getAppEndpoint() + "/hello")
                 .expect(ResponsePredicate.create(AbstractHttpTest::isHttp2x))
                 .expect(ResponsePredicate.status(Response.Status.OK.getStatusCode()))
                 .send().map(resp -> resp.bodyAsJsonObject())
                 .ifNoItem().after(Duration.ofSeconds(TIMEOUT_SEC)).fail()
                 .onFailure().retry().atMost(RETRY);
 
         content.subscribe().with(body -> {
-            assertEquals(body.getString("content"),"Hello, World!");
+            assertEquals(body.getString("content"), "Hello, World!");
             done.countDown();
         });
 
         done.await(TIMEOUT_SEC, TimeUnit.SECONDS);
-        assertTrue(done.getCount() == 0);
+        assertThat(done.getCount(), equalTo(0L));
     }
 
     @Test
     @DisplayName("Http/2 Client Sync test")
     @Disabled("blocked by: https://issues.redhat.com/browse/QUARKUS-658")
-    public void http2ClientSync() {
-        Response resp = httpVersionClientService.getClientHttpVersion();
+    public void http2ClientSync() throws Exception {
+        HttpVersionClientService versionHttpClient = HttpVersionClientServiceBuilder.build(getAppEndpoint())
+                .withHostVerified(true)
+                .withPassword(PASSWORD)
+                .withKeyStorePath(KEY_STORE_PATH)
+                .build();
```

Comment:
The method names of this pattern are weird: HttpVersionClientServiceBuilder.build()....build();
HttpVersionClientServiceBuilder.withHost(getAppEndpoint()), ... .build() is a more natural choice.